supervisor/homeassistant/api.py

```diff
@@ -17,6 +18,8 @@
 
 _LOGGER: logging.Logger = logging.getLogger(__name__)
 
+GET_CORE_STATE_MIN_VERSION: AwesomeVersion = AwesomeVersion("2023.8.0")
```

Comment:
```suggestion
GET_CORE_STATE_MIN_VERSION: AwesomeVersion = AwesomeVersion("2023.8.0dev0")
```

We can change this once the linked core PR merges and is published in nightly

Reply:
The linked core PR has been merged.

Reply:
thanks